rllib/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -376,49 +363,6 @@ def training_step(self) -> ResultDict:
         # Return all collected metrics for the iteration.
         return train_results
 
-    @staticmethod
```

Comment:
**question**: Are we removing execution plans in this PR? why not create a separate PR for it? Always isolate the PRs.

Reply:
This was actually removed in https://github.com/ray-project/ray/pull/25624, which got reverted but somehow that revert seemed to have ended up in a bad merge by me. Thanks for noticing! :)